random-ipv4addrs.py

```diff
@@ -1,19 +1,13 @@
 #!/usr/bin/env python3
 
-import argparse, random, socket, struct
-
-def long2ipv4addr(num):
-    """
-    Convert a long int to a dotted decimal IPv4 address
-    """
-    return socket.inet_ntoa(struct.pack("!L", num))
+import argparse
+from random import getrandbits
+import socket
+import struct
 
 parser = argparse.ArgumentParser()
 parser.add_argument('-c', type=int, default=10000000)
 args = parser.parse_args()
 
-# TODO: verify we have an integer in the input
-
 for i in range(0, args.c):
```

Comment:
You did it! Nice work.  This is even faster with Python2 of course.  I would suggest one change, which I should have done originally, let me know what you think.  change the 'i' to '_' in the loop to make it clear we do not use the loop variable.  Cool?

Reply:
Happy to do it!